File: Java/bee 1011.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        double pi = 3.14159;
        double raio = sc.nextDouble();

        System.out.printf("VOLUME = %.3f\n", (4.0/3.0 * (pi * Math.pow(raio,3))));
    }
}

File: Kotlin/bee 1011.kt
import java.io.BufferedReader
import java.io.InputStreamReader
import kotlin.math.pow

fun main(args: Array<String>){
    val br = BufferedReader(InputStreamReader(System.`in`))
    val pi = 3.14159
    val raio = br.readLine().toDouble()

    println(String.format("VOLUME = %.3f", (4.0/3) * pi * raio.pow(3)))
}
